Person revert

demaris opened the pull request:
reverting after discussion and review.  It looked like redefinition might fix a lot of problems but it seems like they will have to be addressed at level of aiml templates using person tag that should be using person2 tag.

demaris committed:
fix first third person substitution bug

demaris committed:
added comment that the person and person2 tags now match the comment

demaris committed:
switch to previous behavior in hopes of matching majority of used aiml rule set

demaris committed:
switch to previous behavior in hopes of matching majority of used aiml rule set, document inconsistency of  comments with AIML 1.0 spec

wenwei-dev commented:
@demaris This PR didn't revert the last change that swapped person and person2.

wenwei-dev commented:
FYI, I've already reverted "persion fix". #171

demaris commented:
Strange I thought I did it already with the commit yesterday (not a revert,
just changed the variable names back).  The code now looks the same as the
code in my PyCharm without having done a pull.

On Mon, Aug 22, 2016 at 3:17 AM, Wenwei notifications@github.com wrote:

> FYI, I've already reverted "persion fix". #171
> https://github.com/hansonrobotics/HEAD/pull/171
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/hansonrobotics/HEAD/pull/169#issuecomment-241343133,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AF3yl8ZS70EyX58z5dzV-kwWOI_dmQtGks5qiVsZgaJpZM4JpINY
> .

demaris commented:
I'm using a new version of pycharm which maybe works a bit differently.

 I can see two commits, but somehow the only one that got pushed is the
comment change, not the actual code change.  I'd really like to understand
what happened here and what branch if any the actual code change I made is
committed on.

  It definitely works differently as there used to be an option to make a
new branch at the time of push, that doesn't exist now.  I think I expected
to see this option and had to somehow put a comment to force pycharm to see
any change.  Then sometimes it's giving an option to squash all commits to
one, but this time I think it didn't happen, maybe that's the source of
this problem but I don't understand why it didn't give the option (I think
I noticed it at the time).

If history compare of two commits I see the code change, but somehow only
the comment got pushed.
If I look at git branch --contains b8b2 (etc, the commit that didn't get
pushed), it is on person_revert as expected.

I thought that normally a push in Pycharm aggregates all the commits on the
current branch to be pushed.  Did it change?

On Mon, Aug 22, 2016 at 3:17 AM, Wenwei notifications@github.com wrote:

> FYI, I've already reverted "persion fix". #171
> https://github.com/hansonrobotics/HEAD/pull/171
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/hansonrobotics/HEAD/pull/169#issuecomment-241343133,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AF3yl8ZS70EyX58z5dzV-kwWOI_dmQtGks5qiVsZgaJpZM4JpINY
> .
